src/main/java/uk/gov/hmcts/reform/prl/services/reviewdocument/ReviewDocumentService.java

```diff
@@ -206,11 +250,15 @@ private void updateCaseDataUpdatedWithDocToBeReviewedAndReviewDoc(Map<String, Ob
                 caseDataUpdated.put(REVIEW_DOC, document.getCourtStaffQuarantineDocument());
                 log.info(REVIEW_DOC + " {}", document.getCourtStaffQuarantineDocument());
                 break;
+            case BULK_SCAN:
+                caseDataUpdated.put(REVIEW_DOC, document.getUrl());
+                caseDataUpdated.put(REVIEW_DOC, document.getUrl());
```

Comment:
@yogendraU Why are we setting this twice ?

Reply:
thanks @channa-patil for your observation. i have removed the duplicate one.